examples/lvgldemo/Kconfig

```diff
@@ -6,6 +6,7 @@
 menuconfig EXAMPLES_LVGLDEMO
 	tristate "LVGL Demo"
 	default n
+	select GRAPHICS_LVGL
```

Comment:
should we depends on GRAPHICS_LVGL?

Reply:
I was thinking of that. Let me explore the code and find similar example, so I can probably do it in a same style

Reply:
Done